apps/pie-docs/src/patterns/overlay-patterns/guidance/guidance.md

```diff
@@ -0,0 +1,386 @@
+---
+eleventyNavigation:
+    key: Guidance
+    parent: 'Overlay Patterns'
+    order: 2
+eleventyComputed:
+    zIndexes: "{% include './z-indexes.json' %}"
+shouldShowContents: true
+---
+
+## Decision tree
+
+You can follow this decision tree to choose the most appropriate UI component based on the specific context, importance, and user interaction requirements of your product interface.
+
+{% contentPageImage {
```

Comment:
<img width="370" alt="Screenshot 2024-09-12 at 16 50 43" src="https://github.com/user-attachments/assets/60176682-e8e4-4fe4-92c3-720e2ca11fee">

I don't think this is an acceptable image for mobile/small screens - can we please feed this back to design and get some thoughts? I appreciate mobile isn't our primary demographic, however, we want to champion setting great examples for responsive, accessible UIs

Reply:
The image has been updated